### ISSUE TITLE
create process with netlify's dev team to add new attributes and calls <EOM>

### ISSUE THREAD

**Issue by fool** [netlify/open-api#130]

**Comment by ehmicky** [comment]
@fool, has this process been solved?

**Comment by fool** [comment]
nope.  Or at least, if it has, nobody has mentioned to me that it is now a step in the "releasing/changing API endpoints" process and I'd bet it wasn't added there (if there is even a there, there).

It's probably worth taking up as adding to to the project template we're using now in notion though.

**Comment by ehmicky** [comment]
Yes I agree. It seems to me the OpenAPI definition has few discrepancies. Having a process for updating it might be a good way to ensure our clients and the API documentation are in sync with the API.

**Comment by verythorough** [comment]
Yes! As we keep moving towards more "docs-driven devlopment", we're hoping to have it be part of the API dev process to PR changes to the OpenAPI definition _before_ working on changes.

(Of course, we'll also need some sort of process for finding and correcting existing gaps.)

**Comment by fool** [comment]
@verythorough - great to hear!

The process you mention is "just" someone making time for this:

https://github.com/netlify/open-api/issues/131

which I realize doesn't say anything new but it's effectively "audit the
code"

**Comment by verythorough** [comment]
Yeah, I suppose "process" over-sells it a bit. 😛
